UserSettings/EditorUserSettings.asset

```diff
@@ -6,31 +6,34 @@ EditorUserSettings:
   serializedVersion: 4
   m_ConfigSettings:
     RecentlyUsedSceneGuid-0:
-      value: 5a5757560101590a5d0c0e24427b5d44434e4c7a7b7a23677f2b4565b7b5353a
+      value: 520900075c07500a5b5a5b7415220f44404e4b7c742a27612e2b1f60e7e4326c
```

Comment:
I published a new commit to ignore whole "UserSettings" directory, because these are our personal settings. Please rebase your branch, so that this change is properly ignored by Git.

Reply:
Done